test/unit/ngsiv2/expressions/expressionBasedTransformations-test.js

```diff
@@ -193,7 +193,7 @@ var iotAgentLib = require('../../../../lib/fiware-iotagent-lib'),
         providerUrl: 'http://smartGondor.com'
     };
 
-describe('Expression-based transformations plugin', function() {
+describe('NGSI-v2 - Expression-based transformations plugin', function() {
```

Comment:
Not sure but many of these tests are "parallel" to existing ones in NGSIv1 (maybe @dcalvoalonso could confirm, as he did great part of the implementation).

In that case, maybe it's a good idea to add a `'NGSI-v1 - ` prefix also in those ones, for the sake of completeness.

Reply:
Fixed  718ca29